### ISSUE TITLE
Review the French translations

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1071]
While I am pretty confident about the Spanish translations, as it's a language I work with regularly, I'm aware the French translations are clunky in a number of places, but I don't have the fluency and technical vocab to correct them. I admit to having some help from DeepL...

@kelson42 @Popolechien, while it's essential to use translatewiki for other languages in the future, in the interests of making an "anniversary" v4.0.0 release sooner rather than later with French and Spanish, would one of our native French speakers be able to read through the app and send me corrections of anything obviously bad or incomprehensible or horribly clunky? They wouldn't need to edit the actual translation strings. Just copying the visible text on the "Configrurer" and "Informations" pages into a Word / OpenOffice document and marking up would be extremely helpful, or even just making a list in this issue if there are not too many problems.

It would probably take someone 30 mins max that way. I could offer to reciprocate for Spanish (or for English).

**Comment by Popolechien** [comment]
@Jaifroid Happy to do it but I don't see the Kiwix-JS strings on Translatewiki (see image)
![Capture d’écran 2023-08-25 à 08 51 46](https://github.com/kiwix/kiwix-js/assets/21675885/659ac720-fc4f-4241-90d6-5b839b5c14ab)
...and for a reason I can't explain my version of Kiwix-JS (Chrome) does not pick up that my default settings are in French and displays the English UI (which I guess would make it two new issues to open, but nvm).

Please shoot me the strings on a word doc via slack!

**Comment by Jaifroid** [comment]
Many thanks, @Popolechien. It's not up on translatewiki yet, because I've only just inplemented the translation framework, and I haven't yet asked @kelson42 to put the strings up there, as I'm still in the testing phase with just two languages other than English (French and Spanish).

So, all I need at the moment is a proof-read to catch obvious issues. I'll send you the compiled text in a Word document.

Regarding the other issue: remember this new version hasn't been published to Browser stores yet, so if you're accessing it via your installed browser extension, you won't see the version with translation capability. Instead, you have to visit this link directly in your browser:

https://kiwix.github.io/kiwix-js/dist/www/index.html

Does this pick up that your browser is set to French?

**Comment by Popolechien** [comment]
> Does this pick up that your browser is set to French?
Yes it does!

Had a first look and nothing strikes me as clunky, save for one sentence re:dark mode and the appearance of html non-breaking spaces in the API section
![Capture d’écran 2023-08-25 à 09 40 23](https://github.com/kiwix/kiwix-js/assets/21675885/a23b8aab-0829-4dad-b586-05e7b8fd8a88)

**Comment by Jaifroid** [comment]
Thanks. There's quite a bit more text on the about page. I'm just putting together Word doc.

**Comment by Jaifroid** [comment]
Email sent with Word doc. 🙏
